src/workspace-hack/Cargo.toml

```diff
@@ -85,6 +85,7 @@ lru = { version = "0.12.5" }
 memchr = { version = "2.7.5" }
 mime_guess = { version = "2.0.5" }
 minimal-lexical = { version = "0.2.1", default-features = false, features = ["std"] }
+moka = { version = "0.12.10", features = ["future", "sync"] }
```

Comment:
Why do we depend on moka again now? I think we removed it a while ago for [reasons](https://github.com/MaterializeInc/database-issues/issues/5879). If it's pulled in by hashbrown's new features-usage, maybe we should reconsider if we need those features.